### PR TITLE
Enable has_nonlocal_mta rule for RHEL

### PR DESCRIPTION
**Pull request by marcusburghardt**
#### Description:

This `has_nonlocal_mta` rule complements the following CIS requirement by also checking the listening ports.
RHEL7: 2.2.16 - Ensure mail transfer agent is configured for local-only mode (Automated)
RHEL8: 2.2.17 - Ensure mail transfer agent is configured for local-only mode (Automated)
RHEL9: 2.2.15 - Ensure mail transfer agent is configured for local-only mode (Automated)

#### Rationale:

More precise assessment for CIS requirements in RHEL.

### CODE REVIEW

**Review by jan-cerny — APPROVED** [review]

**Comment by marcusburghardt** [comment]
The `testing-farm:centos-stream-8-x86_64` test is failure is not related to this PR.
Due to an issue in the environment test, the compose is using the outdated `scap-security-guide-0.1.63-5.el8.noarch` package. This can be safely waived in this PR.

**Comment by jan-cerny** [comment]
@marcusburghardt Yes, I'm aware of it, but the job is tagged as "Required" which blocks the merge button :cry:

**Comment by marcusburghardt** [comment]
> @marcusburghardt Yes, I'm aware of it, but the job is tagged as "Required" which blocks the merge button cry

The last round passed. : ) The merge button is available now.

**Pull request merged**
